config/v1/types_console.go

```diff
@@ -35,3 +38,29 @@ type ConsoleList struct {
 	metav1.ListMeta `json:"metadata,omitempty"`
 	Items           []Console `json:"items"`
 }
+
+type Auth struct {
+	// configmap reference in the openshift-config namespace
+	// The console needs a CA bundle to validate the certificate used to
+	// talk to the OAuth server.
+	OAuthEndpointCAFile ConfigMapReference `json:"oauthEndpointCAFile"`
+	// If authenticating with Request Header, OAuth or OpenID identity providers,
+	// an external URL is required to destroy single sign-on sessions.
```

Comment:
`LogoutRedirect` is going to remain as the only item on our `.Spec`, but will still be nested under `.Spec.Auth` as this set of config may grow in time.

Reply:
Mark optional and what does `OAuth` mean in 

> Request Header, OAuth or OpenID

Reply:
Updated.